templates/data/generic_component_data.yaml

```diff
@@ -222,32 +222,6 @@ templates:
                     global:
                       - "jobConfig_postsubmit"
                       - "disable_testgrid"
-      - to: ../../prow/jobs/kyma/components/busola-migrator/busola-migrator-generic.yaml
-        jobConfigs:
-          - repoName: "github.com/kyma-project/kyma"
```

Comment:
You are removing the build definition for busola-migrator component that is inside the `kyma` repository. Is this intended and this component is going to be removed, or it's not intended?

Reply:
it is intended.
The PR removing it from kyma will follow
https://github.com/kyma-project/kyma/pull/15414